Workbooks/LogAnalytics Workspace/AtScale/Workspace Insights At Scale.workbook

```diff
@@ -0,0 +1,451 @@
+                  },
+                  {
+                    "columnMatch": "Billable",
+                    "formatter": 3,
+                    "formatOptions": {
+                      "palette": "blue"
+                    },
+                    "numberFormat": {
+                      "unit": 2,
+                      "options": {
+                        "style": "decimal"
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "last_log",
+                    "formatter": 8,
+                    "formatOptions": {
+                      "palette": "greenRed"
+                    },
+                    "numberFormat": {
+                      "unit": 24,
+                      "options": {
+                        "style": "decimal"
+                      }
+                    }
+                  }
+                ]
+              }
+            },
+            "name": "Usage"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "SelectedWS",
+        "comparison": "isNotEqualTo",
+        "value": ""
+      },
+      "name": "Workspace Summary"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "workspace('{SelectedWS}').Operation\r\n| where OperationStatus in (\"Warning\", \"Error\")\r\n| summarize arg_max(TimeGenerated, *) by OperationStatus, OperationCategory, Solution, Computer, Detail\r\n| project TimeGenerated, OperationStatus=tolower(OperationStatus), OperationCategory, Solution, Computer, Detail\r\n| sort by TimeGenerated asc",
+        "size": 3,
+        "title": "Operational errors and warnings",
+        "timeContext": {
+          "durationMs": 0
+        },
+        "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{Workspace}"
+        ],
+        "visualization": "table",
+        "gridSettings": {
+          "formatters": [
+            {
+              "columnMatch": "OperationStatus",
+              "formatter": 11
+            }
+          ],
+          "hierarchySettings": {
+            "treeType": 1,
+            "groupBy": [
+              "OperationStatus",
+              "OperationCategory"
+            ],
+            "expandTopLevel": true
+          },
+          "labelSettings": [
+            {
+              "columnId": "TimeGenerated",
+              "label": "Time"
+            },
+            {
+              "columnId": "OperationStatus",
+              "label": "Status"
+            },
+            {
+              "columnId": "OperationCategory",
+              "label": "Category"
+            },
+            {
+              "columnId": "Detail",
+              "label": "Details"
+            }
+          ]
+        }
+      },
+      "customWidth": "70",
+      "name": "Operation"
```

Comment:
Should this and Machine Sending Heartbeats be conditionally visible? if so maybe we can just move it in the Workspace Summary group?

![image](https://user-images.githubusercontent.com/7664931/102525415-1131c300-404f-11eb-86e9-33db1fedd5c1.png)